src/Interpreters/Aggregator.cpp

```diff
@@ -1204,8 +1204,9 @@ void NO_INLINE Aggregator::executeImplBatch(
                     inst->state_offset,
                     [&](AggregateDataPtr & aggregate_data)
                     {
-                        aggregate_data = aggregates_pool->alignedAlloc(total_size_of_aggregate_states, align_aggregate_states);
+                        AggregateDataPtr place = aggregates_pool->alignedAlloc(total_size_of_aggregate_states, align_aggregate_states);
```

Comment:
Shouldn't we pass place into `createAggreateStates()`?

Reply:
Yes. Will fix